<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrainsVideoPlayer</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutTop">
      <item>
       <widget class="QGraphicsView" name="graphicsView_3">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_4">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutBottom">
      <item>
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView_1">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <item>
       <widget class="QPushButton" name="play_button">
        <property name="text">
         <string>Воспроизведение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pause_button">
        <property name="text">
         <string>Пауза</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restart_button">
        <property name="text">
         <string>Рестарт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="step_backward_button">
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="step_forward_button">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="speed_combobox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>1x</string>
        </property>
        <item>
         <property name="text">
          <string>1x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.2x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10x</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
